src/process/enumeration.rs

```diff
@@ -0,0 +1,43 @@
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
+
+use crate::api_map::APIMap;
+
+#[derive(Debug, Clone)]
+pub struct EnumerationMember {
+    pub identifier: String,
+    pub value: Option<String>,
+}
+
+#[derive(Debug, Clone)]
+pub struct Enumeration {
+    pub identifier: Option<String>,
+    pub r_type: String,
```

Comment:
I usually just use kind instead of r#type, the code looks kind of cleaner without the #